sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -78,7 +87,7 @@ def __init__(self, topic):
       topic: Cloud Pub/Sub topic in the form "/topics/<project>/<topic>".
     """
     super(WriteStringsToPubSub, self).__init__()
-    self._sink = _PubSubPayloadSink(topic)
+    self._sink = _PubSubPayloadSink(coders.BytesCoder(), topic)
```

Comment:
I think it might be wise to explicitly set this and not make people think that it is actually configurable. We can say that we are using this coder explicitly for portability reasons of PubsubIO across implementations in languages.

Also, we assume that the PCollection element type is bytes within ReadStringsFromPubsub and WriteStringsToPubsub.